rdr_service/services/ancillary_studies/nph_incident.py

```diff
@@ -0,0 +1,61 @@
+import logging
+from rdr_service import clock, config
+from rdr_service.services.slack_utils import SlackMessageHandler
+
+from rdr_service.model.study_nph import Incident as NphIncident
+from rdr_service.dao.study_nph_dao import NphIncidentDao
+
+
+_logger = logging.getLogger("rdr_logger")
+
+
+def get_slack_message_handler() -> SlackMessageHandler:
+    slack_config = config.getSettingJson(config.NPH_SLACK_WEBHOOKS, {})
+    if slack_config is None:
+        _logger.error("'slack_config' for 'NPH_SLACK_WEBHOOKS' is empty")
+
+    webbook_url = slack_config.get('nph_incident_alerts', None)
+    if webbook_url is None:
+        _logger.error("'nph_incident_alerts' is not available in slack config. 'webhook_url' is None")
```

Comment:
Make this a warning also, if we have `error` and we process too many errors it will bubble up into the `rdr-alerts` channel.
ex: `logging.warning('Webhook not found. Skipping slack notification for validation error.')`